### PR TITLE
Fields for heat meter added

### PR DESCRIPTION
**Pull request by darshanjain12**
Field for Δt calculated

### CODE REVIEW

File: ontology/yaml/resources/METERS/entity_types/HM.yaml [hunk 1]
```diff
@@ -48,6 +48,9 @@ HM_CHW_WAC:
   is_canonical: true
   uses:
   - water_volume_accumulator
+  opt_uses:
+  - exchange_thermal_temperature_sensor
```
Comment (tasodorff) [inline ontology/yaml/resources/METERS/entity_types/HM.yaml:52]: What is the context for this? exhange thermal?

Comment (darshanjain12) [reply ontology/yaml/resources/METERS/entity_types/HM.yaml]: It is actually a temperature difference between supply water and return water in chilled water pipeline.
Can it be differential temperature sensor

Comment (darshanjain12) [reply ontology/yaml/resources/METERS/entity_types/HM.yaml]: Did the changes

File: ontology/yaml/resources/METERS/entity_types/HM.yaml [hunk 2]
```diff
@@ -48,6 +48,9 @@ HM_CHW_WAC:
   is_canonical: true
   uses:
   - water_volume_accumulator
+  opt_uses:
```
Comment (cstirdivant) [inline ontology/yaml/resources/METERS/entity_types/HM.yaml:51]: Please put these `opt_uses` on the original `HM_CHW` type

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/METERS/entity_types/HM.yaml (hunk 2 above)
Comment (cstirdivant) [reply ontology/yaml/resources/METERS/entity_types/HM.yaml:55]: please remove from here now, it is redundant

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/METERS/entity_types/HM.yaml [hunk 3]
```diff
@@ -39,6 +39,10 @@ HM_CHW:
   - chilled_return_water_temperature_sensor
   - cooling_thermal_power_sensor
   - cooling_thermal_energy_accumulator
+  opt_uses:
+  - differential_temperature_sensor
+  - failed_alarm
+  - water_volume_accumulator
```
Comment (cstirdivant) [inline ontology/yaml/resources/METERS/entity_types/HM.yaml:45]: remove this field, since we have `HM_CHW_WAC` that uses this as optional

**Review by cstirdivant — CHANGES_REQUESTED** [review]

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**